pytorch_lightning/loggers/base.py

```diff
@@ -101,11 +101,6 @@ def update_agg_funcs(
         if agg_default_func:
             self._agg_default_func = agg_default_func
 
-    @property
-    @abstractmethod
-    def experiment(self) -> Any:
```

Comment:
@carmocca Should we have a depreciation there?

Reply:
This was an `abstractmethod` so all subclasses were forced to override it.

Which means that removing it from the parent should be backwards compatible as each subclass will still have it available.